grep.c

```diff
@@ -0,0 +1,48 @@
+#include<stdio.h>
+#include<string.h>
+#include<stdlib.h>
+
+int main (int argc, char *argv[]) {
+    FILE *fp = NULL;
+    char buffer[20];
+    char ch;
+    int count = 0;
+    int index = 0;
+    int iterator;
+    
+    if (argc != 3) {
+        printf("Invalid argument\n");
+	printf("Enter in following format\n");
+	printf("./a.out pattern filename\n");
```

Comment:
Where this pattern used?

Reply:
pattern used in else if statement to compare with readed work in like file.txt